helm/trino-gateway/templates/deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
           # Include the version of trino-gateway-configuration as an input to the deployment checksum. This causes pods to restart on helm update
```

Comment:
Can you move the docs to also use two space indent. Also can you make the comment line shorter .. maybe 80 char width or so